.github/workflows/tests.yml

```diff
@@ -1,8 +1,8 @@
 name: Tests
 
 on:
+  push:
```

Comment:
This should remain running on Pull requests, this ensures new PRs pass the tests

Reply:
Hi @wagslane Oh yes, I mixed things up while providing the PR but also make it available while the changes are not in your repository. I will clean this up and provide the PR again.